### DISCUSSION TITLE
Escaping code nodes

### DISCUSSION THREAD

**Discussion by neongreen** [ueberdosis/tiptap#1867] [General Questions] [answered]
When you create a code node with `` `backticks` ``, if you move the cursor back into the node, you will get stuck inside — I haven't found a way to get out of the node without using the Ctrl+E shortcut.

CKEditor has an option for escaping code nodes by pressing left/right arrow twice when at the beginning/end of a code node. Does tiptap has an option for doing something similar? Or maybe there's another solution?

**Comment by neongreen** [comment]
Noticed that [Remirror](https://remirror.io/) has a clever solution — they add a space whenever you try to escape a code block from the right side and it's the end of the paragraph. I might look there and figure out how they did it.

**Comment by philippkuehn** [comment, marked as answer]
There is nothing built in (for now) but you can look at #1200, #1204 and [here](https://www.tiptap.dev/experiments/trailing-node) for some solutions that may fit your needs.

**Reply by neongreen** [reply]
Oh, the first link is exactly what I need. Thanks!
